ketcher-autotests/tests/File-Management/Open-And-Save-Files/Image-files/image-files.spec.ts

```diff
@@ -891,6 +1066,69 @@ test.describe('Image files', () => {
     await takeEditorScreenshot(page);
   });
 
+  test('Verify that images of (PNG, SVG) with elements can be saved to .cdx file with correct coordinates of images after scaling of them and then opened', async () => {
+    /**
+     * Test case: #2209
+     * Description: Images of (PNG, SVG) with elements can be saved to .cdx file with correct coordinates of images after
+     * scaling of them and after that can be loaded from .ket file with correct positions and layer level.
+     */
+    await openImageAndAddToCanvas('Images/image-svg.svg', page);
+    await openImageAndAddToCanvas('Images/image-png.png', page, 200, 200);
+    await selectRing(RingButton.Benzene, page);
+    await page.mouse.click(200, 500);
+    await selectRectangleSelectionTool(page);
+    await takeEditorScreenshot(page);
+    await page.mouse.click(200, 200);
+    const resizeHandle = page.getByTestId('imageResize-bottomRightPosition');
+    await resizeHandle.scrollIntoViewIfNeeded();
+    await resizeHandle.hover({ force: true });
+    await dragMouseTo(500, 500, page);
+    await takeEditorScreenshot(page);
+    await verifyFile(
+      page,
+      'CDX/image-svg-png-after-scaling-expected.cdx',
+      'tests/test-data/CDX/image-svg-png-after-scaling-expected.cdx',
+      FileType.CDX,
+    );
+    const fileContent = await readFileContents(
+      'tests/test-data/CDX/image-svg-png-after-scaling-expected.cdx',
+    );
+    await openPasteFromClipboard(page, fileContent);
+    await pressButton(page, 'Open as New Project');
+    await takeEditorScreenshot(page);
+  });
+
+  test('Verify that images of (PNG, SVG) with elements can be saved to .cdxml file with correct coordinates of images after scaling of them and then opened', async () => {
+    /**
+     * Test case: #2209
+     * Description: Images of (PNG, SVG) with elements can be saved to .cdxml file with correct coordinates of images after
+     * scaling of them and after that can be loaded from .ket file with correct positions and layer level.
+     */
+    await openImageAndAddToCanvas('Images/image-svg.svg', page);
+    await openImageAndAddToCanvas('Images/image-png.png', page, 200, 200);
+    await selectRing(RingButton.Benzene, page);
+    await page.mouse.click(200, 500);
+    await selectRectangleSelectionTool(page);
+    await takeEditorScreenshot(page);
+    await page.mouse.click(200, 200);
```

Comment:
Better to wrap page.mouse.click on waitForRender
once takeEditorScreenshot follows after the click - it is (at current moment) safe to use unwrapped mouse.click
but in that particular case - it is definitely unsafe

Since you use mouse click on the canvas often - may be better to create function - clickOnCanvas(x, y)

Reply:
Done.